docs/transpile/ai-transpiler-passes.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: AI transpiler passes
+description: What are the AI transpiler passes and how to use them
+---
+
+# AI transpiler passes
+​
+The AI-powered transpiler passes are experimental passes that work as a drop-in replacement of "traditional" qiskit passes for some selected transpiling tasks. They often produce better results than existing heuristic algorithms (such as lower depth and CNOT count), but are also much faster than optimization algorithms such as Boolean satisfiability solvers. The AI transpiler passes run on the cloud and are available to IBM Quantum Premium Plan users.
+
+
+<Admonition type="note">
+    This is an experimental feature available only to the IBM Quantum Premium Plan.
+    The AI-powered transpiler passes are in alpha release status, subject to change.
+</Admonition>
+
+The following passes are currently available:
+
+**Routing passes**
+ - `AIRouting`: Layout selection and circuit routing
+
+The following passes will be available in Q1 2024.
+
+**Circuit synthesis passes**
+ - `AICliffordSynthesis`: Clifford circuit synthesis
+ - `AILinearFunctionSynthesis`: Linear function circuit synthesis
+ - `AIPermutationSynthesis`: Permutation circuit synthesis
+
+
+## Install the qiskit-transpiler-service package
```

Comment:
Do we need to show users how to install the service again if it's already covered in the other page? Instead maybe just say "make sure the qiskit transpiler service is installed, check out the [OTHER PAGE LINK] for details

Reply:
I'll make this change

Reply:
I do agree, we can do that.

Reply:
done in https://github.com/Qiskit/documentation/pull/459/commits/eb8de862d646262b21c1935d01b3e63a4b1be1c4

I didn't see your comment @abbycross. Sorry